installing/installing_ibm_power/installing-ibm-power.adoc

```diff
@@ -0,0 +1,91 @@
+[id="installing-ibm-power"]
+= Installing a cluster on IBM Power
+include::modules/common-attributes.adoc[]
+:context: installing-ibm-power
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+IBM Power infrastructure that you provision.
+
+[IMPORTANT]
+====
+Additional considerations exist for non-bare metal platforms. Review the information in the
+link:https://access.redhat.com/articles/4207611[guidelines for deploying {product-title} on non-tested platforms] before you install an {product-title} cluster.
+====
+
+.Prerequisites
+
+* Provision
+xref:../../storage/persistent_storage/persistent-storage-nfs.adoc#persistent-storage-nfs[persistent storage using NFS]
+for your cluster. To deploy a private image registry, your storage must provide
+ReadWriteMany access modes.
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* If you use a firewall, you must
+xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configure it to allow the sites] that your cluster requires access to.
++
+[NOTE]
+====
+Be sure to also review this site list if you are configuring a proxy.
+====
+
+include::modules/cluster-entitlements.adoc[leveloffset=+1]
+
+include::modules/installation-requirements-user-infra.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
I don't think we're supposed to have an additional resources section right in the middle of an assembly like this. But I'm not positive.

Reply:
That's the solution Vikram allowed because of the recent directive to not keep any resources inside modules. I don't know where we'd put them otherwise.
But if you mean to put them all at the very end of the assembly, then IMO we might as well get rid of them. They'd be hopelessly disconnected from the place they are relevant to and there would potentially be a big bunch of them together at the end (from the whole assembly content).
So, unless there's a directive to not do this, I'd definitely keep them here.

Reply:
Fair enough!